### ISSUE TITLE
Why the weird escape sequences?

### ISSUE THREAD

**Issue by hmil** [pyserial/pyserial#370]
It is the first time I see a terminal use the control sequence `CTRL+]`, while disabling the usual `CTRL+C` and `CTRL+D` sequences. Is this heritage from some system or was it defined ad-hoc for this project?

Note that the sequence `CTRL+]` is impossible to produce on some keyboard layouts.

**Comment by zsquareplusc** [comment]
I wanted something unusual, so that it does not conflict with keys used by the target. CTRL-C and CTRL-D are sent to the target and are useful if you connect to a Linux board for example.

You can change the key with a command line option. However, I would expect that the default key is somehow possible on any keyboard, but I did not verify that. But on my keyboard i can press CTRL+key on Windows but have to use the modifier key on Linux AltGr+CTRL+key.

**Comment by hmil** [comment]
Makes sense. I found it is actually possible to exit the terminal, the key combo seems tied to the location of the `]` key on a US keyboard rather than the `]` character itself. For instance, on CH_FR layout it's `CTRL+¨`.

Maybe there's a way to make sure the banner shows the correct key combo for your layout?
